gradle/checkstyle.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'checkstyle'
+
+checkstyle {
+    toolVersion = '8.1'
+    ignoreFailures = false
```

Comment:
https://github.com/gradle/gradle/pull/780#issuecomment-262097490
but the actual field will be named 
PR for this change: 
https://github.com/gradle/gradle/commit/e6b5c85494be0f385e5622a9e8ed51a76324024f

tagged with: v4.1.0-milestone-1

latetst release was 4.1 RC2 5 days ago, so i guess we will se it soon :)